change/@azure-msal-angular-ca6d42f4-ff4f-44e2-a3fa-726d0852414e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
This can probably be a none change honestly since we haven't released angular v3 yet

Reply:
Makes sense. Changed. Thanks!